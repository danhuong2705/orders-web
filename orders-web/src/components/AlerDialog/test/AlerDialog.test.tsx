import { cleanup, render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import AlertDiaLog from "../AlerDialog";

afterEach(() => {
  jest.clearAllMocks();
  cleanup();
});

describe("<AlertDiaLog />", () => {
  it("should render correctly", async () => {
    const { container } = render(
      <BrowserRouter>
        <AlertDiaLog
          onClose={() => {}}
          isOpen={true}
          description=""
          onActionDialog={() => {}}
        />
      </BrowserRouter>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
