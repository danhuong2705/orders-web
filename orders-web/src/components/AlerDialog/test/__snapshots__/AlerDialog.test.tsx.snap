// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AlertDiaLog /> should render correctly 1`] = `null`;
